build.gradle.kts

```diff
@@ -84,6 +84,11 @@ tasks.getByName("test", Test::class).apply {
 }
 
 val testIntegration = task<Test>("testIntegration") {
+    testLogging.exceptionFormat =  org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+    testLogging.showStackTraces = true
+    testLogging.showExceptions = true
+    testLogging.showCauses = true
+    testLogging.showStandardStreams = true
```

Comment:
Hacks for Travis :)

Reply:
Not really, I think these are useful on their own.

Reply:
Gradle output should be nearly empty. Only actionables should pop up, e.g. warnings, errors.

See [this green build](https://travis-ci.com/atlassian/jira-actions/jobs/281831631). The stdout is full of noise, because some tests cover the unhappy paths. If someone cares about outputs of a test, they can take a look at the test report. The entire stdout, stderr and stacktrace are not necessary, because the test report has it all. And test reports are much better structured than a text stream on stdout.